pusher-api/user-guides/event-subscriptions.md

```diff
@@ -0,0 +1,445 @@
+Event subscriptions
+=====================
+With event subscriptions, you will be immediately updated of those events that are triggered on your Zettle Go instead of pulling information from other Zettle Go APIs.
+
+* [Understand how events work](#understand-how-events-work)
+    * [Payloads](#payloads)
+* [Plan event subscriptions](#plan-event-subscriptions)
+* [Manage event subscriptions](#manage-event-subscriptions)
+
+## Understand how events work
+The Pusher API provides events for you to listen to certain activities of the Zettle Go app at a working HTTPS endpoint on your server.
+
+After you subscribe to events, when an event is triggered, the Pusher API sends a `POST` request that contains a `payload` field with event information to the HTTPS endpoint in real time.
+
+### Payloads
+The `payload` field is the response body from other APIs, such as the Inventory API. The Pusher API sends `POST` requests with the `payload` field in the following JSON format:
+
+```
+{
+    "organizationUuid" : "<organization uuid>",
+    "messageUuid" : "<UUID v1 based on timestamp and messageId>",
+    "eventName" : "<one of the eventnames>",
+    "messageId" : "<unique UUID of the message>",
+    "payload": {
+      "specific payload of the event"
+    },
+    "timestamp": "<event timestamp in ISO-8601 format>"
+  }
+```
+For example, if you subscribe to the `InventoryTrackingStarted` event, when that event is triggered at the Zettle Go app, you will receive a `POST` request that looks similar to the following:
+
+```json
+{
+  "eventName" : "InventoryTrackingStarted",
+  "organizationUuid" : "0f60dbc0-fab5-11e7-b884-62da5a369555",
+  "messageId" : "52662705-98de-588c-810b-75d274d6fa8b",
+  "payload" : {
+    "organizationUuid" : "0f60dbc0-fab5-11e7-b884-62da5a369555",
+    "productUuid" : "18380ac0-fab5-11e7-94b4-842bd3fbd22c",
+    "created" : {
+      "uuid" : "0f674460-fab5-11e7-a310-0002ebd6a43c",
+      "timestamp" : "2018-01-16T12:02:16.569+0000",
+      "userType" : "USER"
+    }
+  }
+}
+```
+
+For more information on event payloads, see the following table.
+
+<table name="payloadAPITable">
```

Comment:
@psvanstrom  @Sarastro72 Please give us a confirmation on the exact number of events supported as of today in the Pusher service. We left placeholders here since we do not have that info. 
We suggest that we strike off all events from the documentation that are not applicable to all integrators. Ref to an API service desk ticket that you may want to take a look at: [API-324](https://ext-izettle.atlassian.net/browse/API-324?focusedCommentId=10589&page=com.atlassian.jira.plugin.system.issuetabpanels%3Acomment-tabpanel#comment-10589)

Reply:
Here's my suggestion on the events that we should actively list. What do you think @Sarastro72 ?

PurchaseCreated
InvoiceCreated
OrganizationUpdated
CardPaymentAuthorized
ProductCreated
ProductUpdated
ProductDeleted
InventoryBalanceChanged
InventoryTrackingStarted
InventoryTrackingStopped
ApplicationConnectionRemoved
PersonalAssertionDeleted

These should be removed from the docs:
OrganizationFeatureUpdated - mostly internal, integrators doesn't have any knowledge of our features

Payment events with undocumented payloads, which we might not want to expose (we might lock these down on client_id):
PaymentInitiated
PaymentCreated
PaymentCanceled
CardPaymentInvalid